# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../clockspeed/x-cross.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- clockspeed-0.62/Makefile	2006-01-02 22:00:37.000000000 +0100
+++ clockspeed-0.62/Makefile	2006-01-02 23:04:32.000000000 +0100
@@ -21,7 +21,7 @@
 
 auto_home.c: \
 auto-str conf-home
-	./auto-str auto_home `head -n 1 conf-home` > auto_home.c
+	echo -e "#include \"auto_home.h\"\nchar auto_home[]=\"`head -n 1 conf-home`\";" > auto_home.c
 
 auto_home.o: \
 compile auto_home.c
@@ -153,8 +153,8 @@
 
 hasrdtsc.h: \
 tryrdtsc.c compile load
-	( ( ./compile tryrdtsc.c && ./load tryrdtsc && ./tryrdtsc \
+	( ( ./compile tryrdtsc.c && ./load tryrdtsc \
 	) >/dev/null 2>&1 \
 	&& echo \#define HASRDTSC 1 || exit 0 ) > hasrdtsc.h
 	rm -f tryrdtsc.o tryrdtsc
 
